test/util/server/server.go

```diff
@@ -187,11 +185,11 @@ func setupStartOptions(useDefaultPort bool) *start.MasterArgs {
 }
 
 func DefaultMasterOptions() (*configapi.MasterConfig, error) {
-	return DefaultMasterOptionsWithTweaks(false)
+	return DefaultMasterOptionsWithTweaks()
```

Comment:
Squash both methods together.

Reply:
> Squash both methods together.

that has ripples.  If I have to rebase.

Reply:
Gotcha, we'll be cleaning that piece of code soon-ish.